one-acre-fund/ussd-sms
# Ser 208 register non clients rw

Allow self-registration for Rwanda (renamed test files for camel case notation)
can be reviewed through commits

To test:

Enter 0 on the first menu to register
Enter a national Id: 119907004198065(any 16 digit)
Enter the first name
Enter the second name
Enter a phone number 078618098(any RW phone)
Enter a group code 0164602302345
Two SMS should be sent containing the account number

Test link: https://telerivet.com/p/8799a79f/service/SV1161debc042a9de0/edit
